src/views/Referrals/Referrals.js

```diff
@@ -124,13 +138,59 @@ export default function Referral(props) {
       });
   };
 
+  const merkleDistributor = getContract(chainId, "MerkleDistributor");
+  const merkleDistributorReader = getContract(chainId, "MerkleDistributorReader");
+
   function handleClaim() {
-    // TODO handle claim
+    setIsClaiming(true);
+    trackAction("Button clicked", {
+      buttonName: "Claim rewards",
```

Comment:
should this be Claim referral rewards? I assume claiming trading rewards uses this same thing

Reply:
yeah, probably, idk with the analytics tbh

Reply:
pretty sure its just a string trait so I just added referral in there